Volume/Supporting Views/VolumeMessage.swift

```diff
@@ -11,11 +11,15 @@ import SwiftUI
 enum Message {
     case noBookmarks
     case upToDate
+    case noFollowingHome
+    case noFollowingPublications
 
     var title: String {
         switch self {
-        case .noBookmarks:
+        case .noBookmarks, .noFollowingHome:
             return "Nothing to see here!"
+        case .noFollowingPublications:
+            return "No Followed Publications"
```

Comment:
why is the capitalization different here?

Reply:
Followed the Figma